src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/Models/ImageWithLateSourceViewModel.cs

```diff
@@ -30,11 +31,13 @@ public string SourceUri
 			}
 		}
 
+		public string SourceUriImmediate => FinalSource;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this property type to 'System.Uri'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5084373)